warehouse/warehouse/workflows/publication/__init__.py

```diff
@@ -73,6 +73,7 @@ def __call__(self, *args, **kwargs):
         warehouse.start_listener()
 
         for dataset_id, dataset in warehouse.datasets.items():
+            log_message('info',f'starting job {self.name} for {dataset_id}')
```

Comment:
this should be "starting workflow"

Reply:
Will fix.

Reply:
OK, I think I've got it all - or mostly/sort-of...  Please review!